packages/steps/src/index.vue

```diff
@@ -42,17 +43,22 @@ export default defineComponent({
       validator: (val: string): boolean => ['wait', 'process', 'finish', 'error', 'success'].indexOf(val) > -1,
```

Comment:
Can we use `includes` here? `indexOf` is a bit of old fashioned way for `includes`